src/components/auction/OrderPlacement/index.tsx

```diff
@@ -219,12 +219,18 @@ const OrderPlacement: React.FC<OrderPlacementProps> = (props) => {
     setAttemptingTxn(false)
   }
 
+  const auctioningToken = React.useMemo(() => derivedAuctionInfo?.auctioningToken, [
+    derivedAuctionInfo?.auctioningToken,
+  ])
+
+  const biddingToken = React.useMemo(() => auctioningToken, [auctioningToken])
```

Comment:
Probably, this line is not intended. I don't understand the switch. of bidding and auctioningToken

Reply:
Yes, good catch!.